caliban/docker.py

```diff
@@ -47,6 +47,7 @@
 TF_VERSIONS = {"2.2.0", "1.12.3", "1.14.0", "1.15.0"}
 DEFAULT_WORKDIR = "/usr/app"
 CREDS_DIR = "/.creds"
+CONDA_BIN = "/opt/conda/bin/conda"
```

Comment:
we have to use the full path, since the environment is not activated yet when Docker is running its commands.